plugins/FrameCombiner/src/main/java/org/micromanager/plugins/framecombiner/FrameCombinerConfigurator.java

```diff
@@ -122,10 +133,12 @@ private void initComponents() {
                 .addContainerGap(javax.swing.GroupLayout.DEFAULT_SIZE, Short.MAX_VALUE))
         );
 
-        pack();
+//        pack();
```

Comment:
the short answer is, it reduces the white space in your GUI. Just like compression. PS: i think i didn't add this. This is comes from the Netbean,  but this is typically what we do for a GUI